.github/workflows/ci-deep.yml

```diff
@@ -91,7 +103,7 @@ jobs:
             cache
             out
             out-optimized
-
+  
```

Comment:
what's up with this space

Reply:
dunno what's up with github, maybe a bug on them (similar to `.sol`) 

if you actually got to look at the file, it looks good

https://github.com/sablier-labs/v2-core/blob/96f89cfabea3dc6638f0b64a30cfb44066a95944/.github/workflows/ci.yml#L93-L107

Reply:
@andreivladbrg there's no bug in GitHub. There were actual extra spaces there that you added in this PR. I have removed them via my latest commit.